include/lsst/ctrl/events/EventSystem.h

```diff
@@ -84,26 +77,23 @@ class EventSystem {
     Event* receiveEvent(const std::string& topicName);
     Event* receiveEvent(const std::string& topicName, const long timeout);
 
-    int64_t createOriginatorId();
-    int extractIPId(int64_t identificationId);
-    int extractProcessId(int64_t identificationId);
-    short extractLocalId(int64_t identificationId);
+    LocationID *createOriginatorId();
 
     StatusEvent* castToStatusEvent(Event* event);
     CommandEvent* castToCommandEvent(Event* event);
 
 private:
+    static EventSystem *defaultEventSystem;
```

Comment:
This looks like a singleton kind of thing, which I'm not sure is wise --- the fact that you can get one from anywhere in the code means your code lies about its interfaces and it's more difficult to test.  Do you really need a singleton?